subsys/shell/Kconfig.backends

```diff
@@ -21,6 +21,12 @@ config SHELL_BACKEND_SERIAL
 
 if SHELL_BACKEND_SERIAL
 
+config SHELL_BACKEND_SERIAL_INIT_PRIORITY
```

Comment:
Shouldn't this parameter be configurable in interrupts priority range?

Reply:
I'm not sure what you mean here. This is initialization priority, not interrupt priority. I don't see any bounds applied to other initialization priorities in various Kconfig's.

Reply:
according to `init.h` it  should be in range 0 to 99.

Reply:
I added range clause, but note that it is not used for any other `_INIT_PRIORITY` definition as far as I can see.

Reply:
Correction: there is `MODEM_GSM_INIT_PRIORITY` that has the range specified.